### PR TITLE
Openshift service scope

### PR DESCRIPTION
**Pull request by suehighmoor**
Annual review of OpenShift Service Scope
- How to build a virtual machine with UKCloud for VMware
- UKCloud glossary
- How to configure an L2 VPN
- How to install and configure OpenVPN
- How to set up Cloud GPU Compute for UKCloud for VMware
- How to migrate vApps between virtual data centres
- Synchronous Protection FAQs
- Journaling Protection Service Scope
- Synchronous Protection Service Scope
- How to access syslog data for your advanced gateway
- Invoice and billing FAQs

### CODE REVIEW

File: articles/openshift/oshift-sco.md [hunk 1]
```diff
@@ -114,90 +105,172 @@ Containers in OpenShift by default don't persist data. Every time an application
...
+#### Container native storage
 
-### SSL certificates
+Kubernetes was not designed with stateful applications like databases in mind. For enterprise applications, these systems have non-negotiable business requirements like high availability, data security, disaster recovery, strict performance SLAs and hybrid/multi-cloud operations.
 
-By default, UKCloud will provide 90 day Let's Encrypt SSL certificates for your OpenShift cluster. At the point of deployment, you can to provide your own certificates for use with OpenShift.
+To address the challenge above, UKCloud offers a [Portworx Enterprise](../openshift/oshift-sd-portworx.md) service, assisting with the adoption of Kubernetes for mission critical enterprise and traditional/legacy applications.
```
Comment (stevemul) [inline articles/openshift/oshift-sco.md:147]: I'd remove this line entirely, I think portworx is being retired?

File: articles/openshift/oshift-sco.md [hunk 2]
```diff
@@ -114,90 +105,172 @@ Containers in OpenShift by default don't persist data. Every time an application
...
+#### SSL certificates
 
-### Infrastructure nodes
+By default, UKCloud will provide 90-day Let's Encrypt SSL certificates for your OpenShift cluster. At the point of deployment, you can provide your own certificates for use with OpenShift.
 
-Additional nodes to support OpenShift core services such as OpenShift routers, registry, metric collection and aggregated logging. Each Foundation Pack is deployed with three Infrastructure Nodes. Where required, you can deploy additional Infrastructure Nodes for a flat monthly fee, with each Infrastructure Node providing an additional 2vCPUs and 16GiB RAM for OpenShift supporting services. You cannot use Infrastructure Nodes for containerised workloads.
+#### Container native storage
 
-### SSL certificates
+Kubernetes was not designed with stateful applications like databases in mind. For enterprise applications, these systems have non-negotiable business requirements like high availability, data security, disaster recovery, strict performance SLAs and hybrid/multi-cloud operations.
```
Comment (stevemul) [inline articles/openshift/oshift-sco.md:145]:
```suggestion
To facilitate the deployment of stateful workloads UKCloud for Managed OpenShift offers dynamically provisioned persistent storage. It is vital that any persistent workloads utilise the appropriate storage policy for the workload to ensure no data loss when a container restarts. The customer is responsible for backup and disaster recovery of any data stored in persistent volumes.
```
We should move away from the advice to not run stateful workloads, we can elaborate on this in future but for now perhaps we just say, for persistence use the relevant storage class?

File: articles/openshift/oshift-sco.md (hunk 1 above)
Comment (stevemul) [reply articles/openshift/oshift-sco.md:147]: If it's not retired we could talk about it for DR and for an alternative approach to persistent storage - but I'd suggest we elaborate some more on why.

File: articles/openshift/oshift-sco.md [hunk 3]
```diff
@@ -114,90 +105,172 @@ Containers in OpenShift by default don't persist data. Every time an application
...
+#### Persistent volumes
 
-UKCloud maintains responsibility for routine patching and upgrades of the OpenShift platform in-line with the published [OpenShift life cycle and patch management process](oshift-ref-lifecycle.md).
+In line with current restrictions on the OpenStack service that underpins OpenShift, you can claim/attach 25 additional persistent volume claims (PVCs) to each worker node.
+
+### Published features not currently available
+
+The following features published in the [UKCloud for Managed OpenShift Service Definition](oshift-sd.md) are not currently available:
+
+- Janet, PSN and RLI connectivity
```
Comment (stevemul) [inline articles/openshift/oshift-sco.md:169]: Did we update the SD and therefore not need this line?

**Review by stevemul — APPROVED** [review]
Some recommended changes - I'll mark as approved anyway to enable this to get merged whenever as it's largely all good. Would probably be worth a bit closer review from my team in the near future though.

File: articles/openshift/oshift-sco.md (hunk 3 above)
Comment (suehighmoor) [reply articles/openshift/oshift-sco.md:169]: Service definition still mentions these networks, so I guess we need to leave this in for now

File: articles/openshift/oshift-sco.md (hunk 1 above)
Comment (suehighmoor) [reply articles/openshift/oshift-sco.md:147]: Removed for now

File: articles/openshift/oshift-sco.md (hunk 2 above)
Comment (suehighmoor) [reply articles/openshift/oshift-sco.md:145]: Implemented suggestion - just had to add a comma!

**Review by vrush20 — APPROVED** [review]
Couple of changes made to the Service Scope 
You may like to insert the "Contact Us" link to the website under the How to Buy section
Minor typo fixed on the FAQ

**Pull request merged**